frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -167,6 +181,29 @@ const getTopologyEdgeItems = (dc: K8sResourceKind, resources: K8sResourceKind[])
       });
     }
   });
+
+  _.forEach(edgesFromServiceBinding(dc, serviceBindingRequests), (edge) => {
+    // handles multiple edges
+    const targetNode = _.get(
+      _.find(resources, (deployment) => {
+        const targetFound =
+          _.get(deployment, 'metadata.ownerReferences[0].name') === edge.resourceRef &&
```

Comment:
Why is this not a comparison against the name and kind of the resource? Why the ownerReference?

Reply:
Looking at the examples it seems that they are probably identifying which service to bind to based on the service name and its kind for eg `Database` etc and not based on its `Deployment`.

Reply:
Service
```apiVersion: postgresql.baiju.dev/v1alpha1
kind: Database
metadata:
  name: db-demo
  namespace: service-binding-demo
spec:
  image: docker.io/postgres
  imageName: postgres
  dbName: db-demo
```

SBR

```
apiVersion: apps.openshift.io/v1alpha1
kind: ServiceBindingRequest
metadata:
  name: binding-request
  namespace: service-binding-demo
spec:
  applicationSelector:
    matchLabels:
      connects-to: postgres
      environment: demo
    group: apps.openshift.io
    version: v1
    resource: deploymentconfigs
  backingServiceSelector:
    group: postgresql.baiju.dev
    version: v1alpha1
    kind: Database
    resourceRef: db-demo
```